### ISSUE TITLE
 jQuery(...).datetimepicker is not a function with two Pods forms with 2.7.0-rc-1

### ISSUE THREAD

**Issue by coding-panda** [pods-framework/pods#4540]
Here is the migration code to create the pods:

{"meta":{"version":"2.7.0-rc-1","build":1508179185},"pods":{"12":{"id":12,"name":"page","label":"Page","description":"","type":"post_type","storage":"table","object":"page","alias":"","fields":{"test":{"id":13,"name":"test","label":"Test","description":"","help":"","class":"","type":"wysiwyg","weight":0,"pick_object":"","pick_val":"","sister_id":"","required":"0","unique":"0","wysiwyg_editor":"tinymce","wysiwyg_media_buttons":"1","wysiwyg_oembed":"0","wysiwyg_wptexturize":"1","wysiwyg_convert_chars":"1","wysiwyg_wpautop":"1","wysiwyg_allow_shortcode":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"rest_read":"0","rest_write":"0","rest_pick_response":"array","rest_pick_depth":"2","wysiwyg_repeatable":0,"wysiwyg_allowed_html_tags":""}},"show_in_menu":1,"pfat_enable":"0","pfat_run_outside_loop":"0","pfat_append_single":"append","pfat_filter_single":"the_content","pfat_append_archive":"append","pfat_filter_archive":"the_content","rest_enable":"0","rest_base":"page","read_all":"0","write_all":"0"},"14":{"id":14,"name":"wysiwyg","label":"Wysiwygs","description":"","type":"pod","storage":"table","object":"","alias":"","fields":{"name":{"id":15,"name":"name","label":"Name","description":"","help":"","class":"","type":"text","weight":0,"pick_object":"","pick_val":"","sister_id":"","required":"1","unique":"0","text_repeatable":"0","text_allowed_html_tags":"strong em a ul ol li b i","text_max_length":"255","text_placeholder":""},"created":{"id":16,"name":"created","label":"Date Created","description":"","help":"","class":"","type":"datetime","weight":1,"pick_object":"","pick_val":"","sister_id":"","datetime_format":"ymd_slash","datetime_time_type":"12","datetime_time_format":"h_mm_ss_A","required":"0","unique":"0","datetime_repeatable":"0","datetime_time_format_24":"hh_mm","datetime_allow_empty":"1","datetime_html5":"0","datetime_type":"format","datetime_format_custom":"","datetime_format_custom_js":"","datetime_time_format_custom":"","datetime_time_format_custom_js":""},"modified":{"id":17,"name":"modified","label":"Date Modified","description":"","help":"","class":"","type":"datetime","weight":2,"pick_object":"","pick_val":"","sister_id":"","datetime_format":"ymd_slash","datetime_time_type":"12","datetime_time_format":"h_mm_ss_A","required":"0","unique":"0","datetime_repeatable":"0","datetime_time_format_24":"hh_mm","datetime_allow_empty":"1","datetime_html5":"0","datetime_type":"format","datetime_format_custom":"","datetime_format_custom_js":"","datetime_time_format_custom":"","datetime_time_format_custom_js":""},"content":{"id":18,"name":"content","label":"Content","description":"","help":"","class":"","type":"wysiwyg","weight":3,"pick_object":"","pick_val":"","sister_id":"","required":"0","unique":"0","wysiwyg_editor":"tinymce","wysiwyg_media_buttons":"1","wysiwyg_oembed":"0","wysiwyg_wptexturize":"1","wysiwyg_convert_chars":"1","wysiwyg_wpautop":"1","wysiwyg_allow_shortcode":"0","pick_post_status":["publish"],"admin_only":"0","restrict_role":"0","restrict_capability":"0","hidden":"0","read_only":"0","roles_allowed":["administrator"],"wysiwyg_repeatable":"0","wysiwyg_allowed_html_tags":""}},"show_in_menu":"1","label_singular":"Wysiwyg","pod_index":"name","hierarchical":"0","ui_style":"post_type","ui_actions_enabled":["add","edit","duplicate","delete"],"ui_reorder_field":"menu_order","ui_fields_manage":{"0":"name","2":"modified"}}}}

Basically, I have two forms at the same time with the two Pods I pasted above:
$mypod = pods( 'page' );
echo $mypod->form();
$my1pod = pods( 'wysiwyg' );
echo $my1pod->form();
If I submit the second one, the console will show: TypeError: jQuery(...).datetimepicker is not a function

**Comment by jamesgol** [comment]
I see the problem here and I also see other potential issues with multiple forms on one page.

First off the problem is that after the form is submitted pods_redirect() is called which uses the php die() function and inserts a JS section that does a redirect to the thank you page.  Because of the die the rest of the page isn't loaded, so any of the javascript dependencies that are in the footer aren't loaded.  Probably not a major issue since it is redirecting and it would just be an error in the console and shouldn't cause any visual problem for the user.

The other issue I see is that PodsAPI->process_form() is checking the nonce of the form to see if it matches and returning an error message ('Access denied, please refresh and try again.') when the data POSTed doesn't match.  I doubt the system was intended or tested with multiple forms on a page.

**Comment by jamesgol** [comment]
Can you test out the release/2.7 branch and see if it solves your issue?  It should, but there is still likely a warning displayed above the first form, I'll have to think about how to handle that.

**Comment by coding-panda** [comment]
@jamesgol  Awesome! It has been fixed. Thanks.
